App/Views/Pages/TrayWindowMainPage.xaml

```diff
@@ -249,5 +249,14 @@
 
             <TextBlock Text="Sign out" Foreground="{ThemeResource DefaultTextForegroundThemeBrush}" />
         </HyperlinkButton>
+
+        <HyperlinkButton
+            Command="{x:Bind ViewModel.ExitCommand, Mode=OneWay}"
+            Margin="-12,0"
```

Comment:
```suggestion
            Margin="-12,-8,-12,-5"
```

Makes them have less space between and also less space under which I think looks better